srcpkgs/guvcview/template

```diff
@@ -1,23 +1,27 @@
 # Template file for 'guvcview'
 pkgname=guvcview
-version=2.0.6
-revision=3
-wrksrc="${pkgname}-src-${version}"
+version=2.0.7
+revision=1
+create_wrksrc=yes
 build_style=gnu-configure
 configure_args="--disable-static --disable-debian-menu"
-hostmakedepends="pkg-config intltool"
+hostmakedepends="pkg-config intltool autoconf automake libtool glib-devel"
 makedepends="eudev-libudev-devel gtk+3-devel SDL2-devel ffmpeg-devel libusb-devel
  pulseaudio-devel v4l-utils-devel portaudio-devel gsl-devel"
 depends="desktop-file-utils"
 short_desc="Simple GTK interface for capturing and viewing video from UVC devices"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-3.0-or-later"
 homepage="http://guvcview.sourceforge.net/"
-distfiles="${SOURCEFORGE_SITE}/${pkgname}/${pkgname}-src-${version}.tar.gz"
-checksum=95381cef5ee139e15f90b79d1425cc22bbaae43f87452cdce6674636aff37e85
+distfiles="${SOURCEFORGE_SITE}/${pkgname}/${pkgname}-src-${version}.tar.bz2"
+checksum=f47d51fbd06e44eeaa1a82f4a153c927527c90d94514197a70b2956028cc8ad4
 
-post_extract() {
+post_patch() {
 	sed '31a#include <locale.h>' -i  guvcview/guvcview.c
 	# add missing EOL in this file to fix do_check()
 	echo >> po/POTFILES.in
```

Comment:
Is this still necessary? Would be nice to upstream both.

Reply:
Yes.
Ticket for the missing EOL exists: https://sourceforge.net/p/guvcview/tickets/30/
Unfortunately I don't have a Sourceforge account and I don't intend to create one to open a ticket for the musl fix.